src/hooks/scrapers.js

```diff
@@ -4,10 +4,10 @@ import createRequestObject from './createRequest'
 
 export async function stockxLowestPrice (item, filter) {
   try {
-    const modelName = item.modelName.replace('(W)', '')
+    const model = item.model.replace('(W)', '')
```

Comment:
Can we do this model and sku formatting once in `getItemPrices` then just pass down the modified item? Feel like its better than having it done all over.

Reply:
will be addressed when transferring input formatting to backend